nuxt.config.js

```diff
@@ -2,6 +2,20 @@ require("dotenv").config();
 const pkg = require("./package");
 const axios = require('axios');
 
+const env = {
+  BASE_URL: process.env.BASE_URL || "http://localhost:3000",
+  PORT: process.env.WEB_PORT || "3000",
+  BROWSER_BASE_URL: process.env.BROWSER_BASE_URL || "/",
+  JWT_SECRET: process.env.jwtSecret || "eefiejfirjfr",
+  MONGO_URL: process.env.MONGO_URL || "mongodb://localhost:27017/",
+  DATABASE_NAME: process.env.DATABASE_NAME || "effectindex",
+  HOST: process.env.HOST || "127.0.0.1",
```

Comment:
What is HOST? How is this different from HOSTNAME / why do we need this?
I can see below that it's being used for mongoose, but if so, it should be clear that it's the `MONGO_HOSTNAME` or similar.